behavior-validation/publish-duplicate/README.md

```diff
@@ -0,0 +1,136 @@
+# Disconnect a client sending duplicate messages
+This cookbook is about to disconnect a client when it sends a message similar to the previous one.
+
+
+### Use-case
+> As a developer, I want to disconnect all clients that send duplicate messages.
+
+For this use-case, a behavior policy is required.
+
+
+### Policy
+
+Consider the following behavior policy that disconnects a client if the received published message is similar to the previous one.
+
+This policy uses `System.log` function to log explanatory messages on each state transition and `Mqtt.disconnect` function to disconnect a client if message count goest above 10.
+
+`policy.json`:
+```json
+{
+  "id": "duplicate-policeman",
+  "matching": {
+    "clientIdRegex": ".*"
```

Comment:
Both examples use `.*`. I think it would be nice to have one use a more specific regex to demonstrate how you can match only certain clients + a brief explanation about that in the cookbook.